src/commonStyles.tsx

```diff
@@ -24,6 +24,58 @@ const styles = StyleSheet.create({
     fontSize: FONT_SIZE,
     fontWeight: '700',
   },
+
+  safeArea: {
+    backgroundColor: '#fff',
+  },
+
+  topBarButton: {
+    height: 25,
+    width: 25,
+    marginLeft: 20,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+
+  container: {
+    flex: 1,
+  },
+
+  generalWrapper: {
+    paddingLeft: 20,
+    paddingRight: 20,
```

Comment:
it can be `paddingHorizontal` https://reactnative.dev/docs/layout-props#paddinghorizontal